pynamodb/pagination.py

```diff
@@ -1,5 +1,5 @@
 import time
-from typing import Any, Callable, Dict, Iterable, Iterator, TypeVar, Optional
+from typing import Any, Callable, Dict, Iterable, Iterator, List, Optional, TypeVar
```

Comment:
Remove "List"

Reply:
Thanks. We really ought to upgrade to repo to common tooling (black, reorder-python-imports, pyupgrade) but I'm dreading the diff size 😅